pkg/distributor/distributor.go

```diff
@@ -237,6 +240,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 	cfg.RetryConfig.RegisterFlags(f)
 
 	f.IntVar(&cfg.MaxRecvMsgSize, "distributor.max-recv-msg-size", 100<<20, "Max message size in bytes that the distributors will accept for incoming push requests to the remote write API. If exceeded, the request will be rejected.")
+	f.IntVar(&cfg.MaxOTLPRequestSize, maxOTLPRequestSizeFlag, 80<<20, "Maximum OTLP request size in bytes that the distributors accept. Requests exceeding this limit are rejected.")
```

Comment:
How was the 80 MiB selected? Do we have some data about size of the request and processing time?

Intuitively 80 MiB still feels very large. With previous 100 MiB we sometimes see translation to Mimir request taking up to 10 seconds, so perhaps better value would be somewhere close to 10 MiB?

Reply:
as mentioned in the description of the PR, it is tentative. Since 413 error is not retryable, we need to be cautious. if 100MiB/10s, how about 2s of response time ~ 20 MiB

Reply:
I agree that we need to be careful when rolling this out. It may be safer to use original value of 100 MiB for now, so that this doesn't break existing setups, and then we can experiment with correct value in our setup.

Reply:
let me revert my reverted commit